docs/src/style.md

```diff
@@ -144,8 +144,16 @@ function foo(x)
 end
 ```
 
+#### Line length
 
-#### TODO: Line breaks
+Following the Google style guide for [line length](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length)
+in python, keep lines under an 80 character limit.
```

Comment:
This is a contentious point so we need to give our own motivation and not just defer to the google style guide.
My thinking is that for consistency, we need *some* guideline on line length. 80 characters is a reasonable pre-existing convention.

Reply:
Maybe we just make this a recommendation. What about:

Our foremost goal is to maximize code readability. Very long line lengths are hard to read since they either require horizontal scrolling, or are wrapped uncontrollably by a text editor. Thus, where possible, keep line lengths under 80 characters (an arbitrary, but pre-existing convention). We make an exception for lines in which manually breaking the line over multiple lines is worse for readability.

Reply:
"wrapped uncontrollably by a text editor" is pretty unconvincing

Reply:
yes, please just make it a recommendation. I don't plan to start enforcing line lengths in any packages I author. though for packages that aren't mine, I will follow the surrounding style.

Reply:
@mlubin it is also "consistent" to not have line limits at all